lib/broadway/topology/rate_limiter.ex

```diff
@@ -107,4 +107,12 @@ defmodule Broadway.Topology.RateLimiter do
   defp schedule_next_reset(interval) do
     _ref = Process.send_after(self(), :reset_limit, interval)
   end
+
+  defp get_pid(process_name) when is_atom(process_name) do
+    Process.whereis(process_name)
+  end
+
+  defp get_pid({:via, Registry, {registry, args}}) do
+    registry.whereis_name(args)
+  end
```

Comment:
Not sure if there's a cleaner way to do this – can't rely on `Process.whereis` to find everything any more

Reply:
Saw your comment - TIL!